### PR TITLE
Add notes for Chocolatey Central Management 0.12.0

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for Chocolatey Central Management v0.12.0.

## Motivation and Context

There is a release coming up, and we need to get this documented.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/central-management/release-notes.md [hunk 1]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Provide ability to create a Deployment Plan for a single outdated package on all affected Computers.
+- Provide ability to create a Deployment Plan for all outdated packages on all Computers.
+- Provide ability to create an empty Deployment Plan directly for a specific Computer - see [Licensed #259](https://github.com/chocolatey/chocolatey-licensed-issues/issues/259).
+- Provide ability to create a Deployment Plan for all outdated packages for a single Computer - see [Licensed #281](https://github.com/chocolatey/chocolatey-licensed-issues/issues/281).
+- Provide ability to duplicate an existing Group.
+- Provide ability to create an empty Deployment Plan directly for a specific Group.
+
+### Bug Fixes
+
+- Fix - Certain roles within the Chocolatey Central Management website are able to impersonate other users using the "Login as this user" action menu - see [Licensed #318](https://github.com/chocolatey/chocolatey-licensed-issues/issues/318).
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:46]: This doesn't explain what the fix is. It's a statement of functionality. Can we reword this?

File: input/en-us/central-management/release-notes.md [hunk 2]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Fix - Unnecessary permissions should not be present in the `Edit Roles` and `Permissions` modal windows.
+- Fix - Notification about scheduled Deployment Plan failing to start uses incorrect date format.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:48]:
```suggestion
- Fix - Notification about a scheduled Deployment Plan failing to start uses incorrect date format.
```

File: input/en-us/central-management/release-notes.md [hunk 3]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Fix - Inconsistent presentation of Export and Action buttons across pages.
+- Fix - Missing TimeZone setting for users in Colombia - (UTC - 05:00) Bogota, Lima, Quito, Rio Branco - see [Licensed #356](https://github.com/chocolatey/chocolatey-licensed-issues/issues/356).
+- Fix - The HTTP error page of the Chocolatey Central Management Website does not use standard styling.
+- Fix - Different colors are used for the "yes/no" badges on various tables across the website.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:52]:
```suggestion
- Fix - Ensure colors for the "yes/no" badges are consistent on various tables across the website.
```

File: input/en-us/central-management/release-notes.md [hunk 4]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Fix - Encryption Settings information callout is dismissible when it shouldn't be.
+- Fix - Password reset link no longer displays an "invalid link" message after updating the Encryption Passphrase.
+- Fix - Tables of data do not retain current page when refreshed automatically.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:55]: I don't know what this is when reading it. Can it be reworded?

File: input/en-us/central-management/release-notes.md [hunk 5]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Fix - Names of Deployment Steps can cause an overflow on some modal windows.
+- Fix - Action buttons on tables that only contain one option are hidden.
+- Fix - Existing configuration stored in the `appsettings.json` file are not respected when upgrading either the Chocolatey Central Management Database or Service packages.
+- Fix - Elements on page overlap when a Passphrase is required to be reset by an Admin.
+
+### Improvements
+
+- Provide ability to have a Group Details screen.
+- Provide ability to automatically fetch updated information for main Deployment Plan page.
+- Provide ability to see information about the last refreshed time for KPI Dashboard.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:66]:
```suggestion
- Provide ability to see information about the last refreshed time for Dashboard.
```

File: input/en-us/central-management/release-notes.md [hunk 6]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Provide ability to see the number of Computers that have a particular piece of Software installed - see [Licensed #231](https://github.com/chocolatey/chocolatey-licensed-issues/issues/231).
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:67]:
```suggestion
- Provide ability to see the number of Computers that have specific Software installed - see [Licensed #231](https://github.com/chocolatey/chocolatey-licensed-issues/issues/231).
```

File: input/en-us/central-management/release-notes.md [hunk 7]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Provide ability to duplicate a Deployment Step within a Deployment Plan - see [Licensed #332](https://github.com/chocolatey/chocolatey-licensed-issues/issues/332).
+- Information about the number of ready/active Deployment Plans is available within the Dashboard page.
+- Reporting - Truncate all Excel sheet names to required 31-char limit.
+- Confusing TimeZone options for Central Europe in Chocolatey Central Management - see [Licensed #325](https://github.com/chocolatey/chocolatey-licensed-issues/issues/325).
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:71]: This doesn't fatty what's been done. It's just a statement. Can it be reworded?

File: input/en-us/central-management/release-notes.md [hunk 8]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Add a KPI to the Dashboard page to show the number of Computers that have not recently reported into Chocolatey Central Management.
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:72]:
```suggestion
- Add an item to the Dashboard page to show the number of Computers that have not recently reported into Chocolatey Central Management.
```

File: input/en-us/central-management/release-notes.md [hunk 9]
```diff
@@ -30,6 +30,53 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Shorten generated names for Deployment Steps to remove redundant "Deployment Step" prefix.
+
+### Documentation
+
+- Normalize wording for Deployment Plan and Deployment Step in UI
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:77]:
```suggestion
- Normalize wording for Deployment Plan and Deployment Step in UI.
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
Myself and @pauby went through all of his points in a pairing session, and we are happy to continue with getting this merged.

**Pull request merged**